cmd/oceantv/utils.go

```diff
@@ -162,12 +163,39 @@ func updateConfigWithTransaction(ctx context.Context, store Store, skey int64, b
 	}
 
 	err := store.Update(ctx, key, updateConfig, &model.Variable{})
-	if errors.Is(err, datastore.ErrNoSuchEntity) {
+	// We need to check for both filestore and cloudstore error types here.
+	if errors.Is(err, datastore.ErrNoSuchEntity) || errors.Is(err, googledatastore.ErrNoSuchEntity) {
```

Comment:
Test for openfish/datastore errors, not cloud.google.com/go/datastore errors.

Reply:
This can now be left just as datastore.ErrNoSuchEntity since this PR:
https://github.com/ausocean/openfish/pull/147